manifests/install.pp

```diff
@@ -8,6 +8,7 @@
   $manage_ruby_dependency,
   $puppet_master = true,
   $is_pe_server = $r10k::params::is_pe_server,
+  Optional[String[1]] $gem_source = undef,
```

Comment:
can you please document the new parameter?

Reply:
Sure thing, thanks. All set...